src/package.coffee

```diff
@@ -200,7 +200,12 @@ class Package
 
   activateResources: ->
     @activationDisposables = new CompositeDisposable
-    @activationDisposables.add(atom.keymaps.add(keymapPath, map)) for [keymapPath, map] in @keymaps
+
+    value = not _.include(atom.config.get("core.disabledKeymaps") ? [], @name)
+    if value
+      @activateKeymaps()
+    else if not value
+      @deactivateKeymaps()
```

Comment:
You can remove the second `if`, since its condition is the inverse of the first `if`. How about this:

```coffee
keymapIsDisabled = _.include(atom.config.get("core.disabledKeymaps") ? [], @name)
if keymapIsDisabled
  @deactivateKeymaps()
else
  @activateKeymaps()
```